src/filtrering/filtrering-filter/filterform/geografiskbosted-filterform.tsx

```diff
@@ -0,0 +1,139 @@
+import {FiltervalgModell} from '../../../model-interfaces';
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import classNames from 'classnames';
+import Grid from '../../../components/grid/grid';
+import {Checkbox, CheckboxGroup} from '@navikt/ds-react';
+import NullstillKnapp from '../../../components/nullstill-valg-knapp/nullstill-knapp';
+import {useGeografiskbostedSelector} from '../../../hooks/redux/use-geografiskbosted-selector';
+import {GeografiskBostedOptions} from '../../../ducks/geografiskBosted';
+import {MultiSelect} from 'react-multi-select-component';
+
+interface GeografiskBostedFilterformProps {
+    endreFiltervalg: (form: string, filterVerdi: string[]) => void;
+    filtervalg: FiltervalgModell;
+    gridColumns?: number;
+    emptyCheckboxFilterFormMessage?: string;
+}
+
+function GeografiskBostedFilterform({endreFiltervalg, filtervalg, gridColumns = 1}: GeografiskBostedFilterformProps) {
+    const [visGeografiskBosted, setVisGeografiskBosted] = useState<string[]>(filtervalg.visGeografiskBosted);
+    const [selectedGeografiskBosted, setSelectedGeografiskBosted] = useState<GeografiskBostedOptions[]>([]);
+    const [geografiskBostedOptions, setGeografiskBostedOptions] = useState<GeografiskBostedOptions[]>([]);
+    const geografiskbostedListData = useGeografiskbostedSelector();
+
+    useEffect(() => {
+        setVisGeografiskBosted(filtervalg.visGeografiskBosted);
+
+        if (geografiskbostedListData != null && geografiskbostedListData.size > 0) {
+            let selectedValues: GeografiskBostedOptions[] = [];
+            filtervalg.geografiskBosted.forEach(x => {
+                if (geografiskbostedListData.get(x) != null && geografiskbostedListData.get(x) !== undefined) {
+                    selectedValues.push({label: geografiskbostedListData.get(x)!, value: x, checked: true});
+                }
+            });
+            setSelectedGeografiskBosted(selectedValues);
+        }
+    }, [filtervalg, geografiskbostedListData]);
+
+    const velgGeografiskBosted = data => {
+        endreFiltervalg(
+            'geografiskBosted',
+            data.map(x => x.value)
+        );
+    };
+
+    useEffect(() => {
+        if (geografiskbostedListData != null && geografiskbostedListData.size > 0) {
```

Comment:
Redundant null-check? geografiskBostedListData's type states that it cannot be null.

Reply:
Also this could be "simplified" to `setGeografiskBostedOptions(Array.from(geografiskbostedListData).map((data: [string, string]) => ({label: data[0], value: data[1]})))`.

Reply:
Fixed, thank you!